spec/controllers/forgot_password_controller_spec.rb

```diff
@@ -3,12 +3,11 @@
 RSpec.describe ForgotPasswordController do
   describe '#show' do
     context 'email in session' do
-      it 'renders the page and deletes the email from the session' do
+      it 'renders the page' do
         session[:email] = 'test@example.com'
 
         get :show
 
-        expect(session[:email]).to be_nil
```

Comment:
I think this is still a valid assertion? Since the `.delete` was moved into the view model?